projects/Mallard/src/components/EditionsMenu/RegionButton/RegionButton.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react'
+import { StyleSheet, Text, TouchableWithoutFeedback, View } from 'react-native'
+import { TitlepieceText } from 'src/components/styled-text'
+import { color } from 'src/theme/color'
+import { families } from 'src/theme/typography'
+
+const styles = (selected: boolean) =>
+    StyleSheet.create({
+        container: {
+            backgroundColor: selected ? color.primary : '#E5E5E5',
+            paddingBottom: 32,
+            paddingLeft: 104,
+            paddingTop: 4,
+        },
+        title: {
+            fontSize: 20,
+            lineHeight: 24,
+            marginBottom: 5,
+            color: selected ? 'white' : color.primary,
+        },
+        subTitle: {
+            fontFamily: families.sans.regular,
+            fontSize: 15,
+            lineHeight: 16,
+            color: selected ? 'white' : color.text,
+        },
+    })
+
+const RegionButton = ({
+    onPress,
+    selected = false,
+    subTitle,
+    title,
+}: {
+    onPress: () => void
+    selected?: boolean
```

Comment:
should this be optional given that you have provided a default for it?

Reply:
Will still throw typescript errors if its not defined when its used.